src/master-playlist-controller.js

```diff
@@ -8,6 +8,8 @@ import videojs from 'video.js';
 import AdCueTags from './ad-cue-tags';
 import SyncController from './sync-controller';
 import { translateLegacyCodecs } from 'videojs-contrib-media-sources/es5/codec-utils';
+import work from 'webworkify';
```

Comment:
I think `work` is a bit generic and `worker` would be a more descriptive name for the function.